.github/workflows/umbral-pre.yml

```diff
@@ -130,12 +130,16 @@ jobs:
         working-directory: umbral-pre-python
 
       - name: Install pip dependencies
-        run: pip install mypy
+        run: pip install mypy ruff
 
       - name: Run mypy.stubtest
         run: python -m mypy.stubtest umbral_pre --allowlist stubtest-allowlist.txt
         working-directory: umbral-pre-python
 
+      - name: Run ruff
+        run: ruff check umbral_pre
```

Comment:
```suggestion
        run: ruff check --output-format=github umbral_pre
```

Reply:
See comment here about possible matrix values - https://github.com/nucypher/rust-umbral/pull/130#discussion_r1412160286.

If that comment holds and we use more than one version for the matrix it could make sense to move ruff to a separate job (similar to what we did yesterday in that `nucypher-ops` PR).